### ISSUE TITLE
feat: s3 plugin sqlite format

### ISSUE THREAD

**Issue by guilhermefrag** [cloudquery/cloudquery#14494]
### Which problem is this feature request solving?

I would like to have the option to create a sqlite with the postgres source and send it to s3 destination

### Describe the solution you'd like

Is there a way to merge the sqlite plugin with the s3 plugin?

### Pull request (optional)

- [ ] I can submit a pull request

**Comment by pilvikala** [comment]
@guilhermefrag I would just like to make sure I understand this right: is your end goal a sqlite file created from a source postgres database stored in an S3 bucket?

**Comment by guilhermefrag** [comment]
@pilvikala that's right, exactly

**Comment by erezrokah** [comment]
Hi @guilhermefrag, I think a possible workaround is to use the Postgres source with the SQLite destination then the AWS CLI to upload the SQLite file to S3.

Another more composable approach would be to source from Postgres to SQLite in one sync, then have a separate sync with a generic file source plugin (which we don't have since source plugins usually read data into tables structure so they need to know the file format) to an S3 destination

**Comment by yevgenypats** [comment]
Im going to close this as no-fix/out-of-scope. it's best to work with sqlite locally and then upload once the sync is done.
